public/user_img.html

```diff
@@ -42,13 +42,13 @@
         <div class="collapse navbar-collapse" id="navbarResponsive">
           <ul class="navbar-nav ml-auto">
             <li class="nav-item mx-0 mx-lg-1">
-              <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#portfolio">Home</a>
+              <a href="index.html" class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger">Home</a>
```

Comment:
would have href point to "/" so that it doesn't show up as index.html in the URL